demisto_sdk/commands/validate/tests/GR_validators_test.py

```diff
@@ -529,3 +532,147 @@ def test_IsUsingUnknownContentValidator__different_dependency_type__list_files(
         [repo_for_test.packs[pack_index]]
     )
     assert len(results) == expected_len_results
+
+
+@pytest.fixture
+def repo_test_from_version(graph_repo: Repo):
+    # Repo which contains two packs
+
+    # Pack 1 - playbook uses an integration (relationship)
+    pack_1 = graph_repo.create_pack("Pack1")
+    playbook_using_deprecate_commands = {
```

Comment:
I think the "deprecated" is not needed for this test, and is miss leading. please renmae